_templates/faqs.md

```diff
@@ -28,26 +40,26 @@ pitsid: [A number, representing the public issue tracker ID associated with the
   </tr>
   <tr>
     <td>Mobile development type</td>
-    <td>Hybrid (Cordova app), Native (NativeScript app)</td>
+    <td>Hybrid ({{site.ac}} app), Native ({{site.ns}} app)</td>
   </tr>
   <tr>
-    <td>Code (AppBuilder) client</td>
-    <td>In-Browser, Extension for Visual Studio, Universal Desktop, Windows, Command-line interface, Sublime Text</td>
+    <td>{{site.ab-sab}}<!--Code (AppBuilder)--> client</td>
+    <td>{{site.ab-web}}, {{site.ab-vse}}, {{site.ab-proton}}, {{site.ab-win}}, {{site.ab-cli}}, {{site.ab-sp}}</td>
   </tr>
   <tr>
     <td>Mobile OS</td>
-    <td>iOS, Android, Windows Phone 8</td>
+    <td>iOS, Android, {{site.wp8}}</td>
```

Comment:
Not sure if we need this. Windows Phone is an old product and its name won't change. Additionally, I doubt if we will ever see Windows Universal in AppBuilder :)